### DISCUSSION TITLE
Sword module access Android

### DISCUSSION THREAD

**Discussion by tonhou** [ezra-bible-app/ezra-bible-app#1127] [Q&A] [answered]
I have AndBible installed on Android. Is it possible to access the same Sword modules?
Or can I copy them to a folder that Ezra Bible App can access them?
Thanks

**Comment by tobias-klein** [comment, marked as answer]
Hi @tonhou 
The internal storage paths are not accessible to other apps than the one "owning" them.
Aside from the internal storage path, Ezra Bible App is configured to use (read from) the generic SWORD directory `/sdcard/Documents/sword`. Any modules stored there are also accessible to Ezra Bible App.
I am not sure if AndBible uses an external storage path to store SWORD modules.
You may acquire with the AndBible team.
